frontend/pages/hosts/details/cards/Activity/ActivityItems/InstalledSoftwareActivityItem/InstalledSoftwareActivityItem.tsx

```
@@ -50,8 +50,7 @@ const InstalledSoftwareActivityItem = ({
         isSoloActivity={isSoloActivity}
       >
         <b>{title}</b> {passivePrefix} on this host
-        {from_setup_experience ? " during setup experience" : ""} (self-service)
-        .
+        {from_setup_experience ? " during setup experience" : ""} (self-service).
```

ESLint

[error] Insert `{"·"}⏎·······`

(prettier/prettier)